proposals/0065-collections-move-indices.md

```diff
@@ -472,7 +472,7 @@ in the interest of full disclosure:
   manipulations end up looking like free function calls:
 
   ```swift
-  let j = index(after: i)           // self.successor(i)
+  let j = index(after: i)           // self.index(after: i)
```

Comment:
It's just that the text of the proposal pre-dates the new index method (it used to be called successor).